specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json

```diff
@@ -710,6 +710,253 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/basicPublishingCredentialsPolicies": {
```

Comment:
the resource type name is pretty verbose. Can it be more concise?

Reply:
Unfortunately this API controls some very specific config on a site resource and any shorter version of basicPublishingCredentialsPolicies would not provide an accurate picture of what these resources portray. If we need to refactor this in the future we will consider making it more concise.